src/App/App.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+import USAState from '../USAState/USAState';
+import ComparisonContainer from '../ComparisonContainer/ComparisonContainer';
+import Stat from '../Stat/Stat';
+import './App.scss';
+
+interface Props { 
+}
+
+interface State {
+  selectedUSAState: {};
+}
+
+class App extends Component<Props,State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      selectedUSAState: {}
+    }
+  }
+
+  render() {
+    return(
+      <main>
+        <h1>COCO</h1>
+        <body>
```

Comment:
I think we should add a section here to display USA total stats - it will require an additional API call but will make the page more informative as well as allow us to compare state information to national information.

Reply:
I like that idea!